lensterxyz/lenster
# feat: show time ago timestamp under message tile (#947)

## What does this PR do?


On /messages page, change from showing absolute datetime for a message, to showing "x days/hours/minutes ago".
Also added a title attribute so that the absolute datetime is shown when hovering mouse over the "x ago" label.
Fixes #947

![Screenshot_2022-11-08_18-28-46](https://user-images.githubusercontent.com/667227/200649780-e5413c82-0e96-46fb-8149-9f2c587c623d.png)


## Type of change


## How should this be tested?

Open a message conversation and check that timestamps are shown as e.g. "5 minutes ago"
